packages/server/validators/guilds.js

```diff
@@ -231,11 +298,39 @@ const updateGuildMemberRoleValidator = [
     .blacklist("<>")
     .matches(/^(?:Member|Officer|Owner)$/)
     .withMessage("Invalid role. Allowed values are: Member, Officer, or Owner"),
+  param("guildId", "userId").custom(async (_, { req }) => {
```

Comment:
```js
param("guildId", "userId").custom(async (_, { req }) => {
  // ...
});
```

Same thing as my other comment about defining an express-validator for multiple fields. These should be in an array as the first argument.
```js
param(["guildId", "userId"])
```